README.md

```diff
@@ -103,49 +103,46 @@ pytest
 
 ### From Command Line
 
-To launch the framework with the GUI
-
+To launch the framework with the GUI:
 ```bash
-aidesign
+vai_lab
 ```
 
-or to execute an existing config file
-
+To launch the framework with an existing config file
 ```bash
-aidesign --file <path_to_config_file>
+vai_lab --file <path_to_config_file>
 ```
 
 
 ### From Code
 
-To launch the framework with the GUI
+To launch the framework with the GUI:
 
 ```python
-import aidesign as ai
+import vai_lab as ai
 
 core = ai.Core()
 core.run()
 ```
 
-or to execute an existing config file
+or to execute an existing config file:
 
 ```python
-import aidesign as ai
+import vai_lab as ai
 
 core = ai.Core()
 core.load_config_file("<path_to_config_file>")
 core.run()
 ```
 
-
 ### Examples
 
 Pre-made [examples](https://github.com/AaltoPML/ai-assisted-framework-design/tree/main/aidesign/examples/xml_files) show the syntax and form of the config files and pipeline as a whole, and are the best way to get started.

```

Comment:
aidesign -> src/vai_lab

```suggestion
Pre-made [examples](https://github.com/AaltoPML/ai-assisted-framework-design/tree/main/src/vailab/examples/xml_files) show the syntax and form of the config files and pipeline as a whole, and are the best way to get started.

```

Reply:
Good catch - forgot I was going through my grep list! Changed these and all other refs to ai_design I could find - thanks!